### PR TITLE
[DO NOT MERGE] First bootstrap 4 attempt

### PR DESCRIPTION
**Pull request by OmgImAlexis**
@MGaetan89 you'll probably need to still fix a LOT of the CSS but for the most part this is working. @VinceVal and @aarondunlap said they'd help where they can.

### CODE REVIEW

**Comment by SiCKRAGEB0T** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your report has all the below criteria checked

- [x] Give a description on what the PR is for.
- [x] Make sure you send a PR against the DEVELOP branch
- [ ] Don't send big changes in one go. Split up big PRs into multiple smaller (easier manageble) PRs

Please make sure you also read [contribution guide](https://github.com/SiCKRAGETV/SickRage/blob/master/contributing.md) and followed all the steps.

Thanks!

File: gui/slick/views/layouts/main.mako [hunk 1]
```diff
@@ -64,283 +46,183 @@
...
-        <link rel="icon" type="image/png" sizes="16x16" href="${srRoot}/images/ico/favicon-16.png">
-        <link rel="apple-touch-icon" sizes="152x152" href="${srRoot}/images/ico/favicon-152.png">
-        <link rel="apple-touch-icon" sizes="144x144" href="${srRoot}/images/ico/favicon-144.png">
-        <link rel="apple-touch-icon" sizes="120x120" href="${srRoot}/images/ico/favicon-120.png">
-        <link rel="apple-touch-icon" sizes="114x114" href="${srRoot}/images/ico/favicon-114.png">
-        <link rel="apple-touch-icon" sizes="76x76" href="${srRoot}/images/ico/favicon-76.png">
-        <link rel="apple-touch-icon" sizes="72x72" href="${srRoot}/images/ico/favicon-72.png">
-        <link rel="apple-touch-icon" href="${srRoot}/images/ico/favicon-57.png">
+        <!-- Bootstrap CSS -->
+        <link rel="stylesheet" type="text/css" href="https://v4-alpha.getbootstrap.com/dist/css/bootstrap.min.css">
```
Comment (MGaetan89) [inline gui/slick/views/layouts/main.mako]: Isn't it best to use locale file?

File: gui/slick/views/layouts/main.mako [hunk 2]
```diff
@@ -19,30 +19,12 @@
...
-        <meta name="viewport" content="width=device-width">
-
-        <!-- These values come from css/dark.css and css/light.css -->
-        % if sickbeard.THEME_NAME == "dark":
-        <meta name="theme-color" content="#15528F">
-        % elif sickbeard.THEME_NAME == "light":
-        <meta name="theme-color" content="#333333">
-        % endif
-
-        <title>SickRage - BRANCH:[${sickbeard.BRANCH}] - ${title}</title>
```
Comment (MGaetan89) [inline gui/slick/views/layouts/main.mako]: Where did lines 29 to 36 go?

File: gui/slick/views/layouts/main.mako (hunk 1 above)
Comment (OmgImAlexis) [inline gui/slick/views/layouts/main.mako]: This is just to show how everything needs to be moved to and currently there's no downloadable version of it and I haven't setup grunt for Sass which I hope we move to as it'll allow theming a lot easier than CSS.

File: gui/slick/views/layouts/main.mako (hunk 2 above)
Comment (OmgImAlexis) [inline gui/slick/views/layouts/main.mako]: They need to be added back, check the above comment.

**Comment by OmgImAlexis** [comment]
Since there's loads of bootstrap v4 that still is changing this will be closed for now.

**Pull request closed without merging**